JS/popupPage.js

```diff
@@ -112,12 +111,17 @@ projectBtn.forEach((btn, btnIndex) => {
         </li>
       </ul>
     </div>
+  </div>
   </div>
     `;
 
+    projectPopup.style.display = 'block';
```

Comment:
- [ ] The asked changes are not implemented yet, please take advice from what was said in the previous review and fix the code once more, check the record below
- Your `live source and see demo` button is not sharing the same color as per the original design, please update that one to
- Some of the card does not open the pop up
- all your pop-up modal does not have the image showing


![5](https://github.com/GeekyHacks/My-Personal-Portfolio/assets/77843050/281197a9-dd80-4a2f-b7ad-aa5ab5fd7770)

Reply:
https://github.com/GeekyHacks/My-Personal-Portfolio/assets/86020057/2a0bb0b9-117c-40d7-89ea-fc811d291632